src/en/users/getting-started/index.html

```diff
@@ -124,19 +89,23 @@ <h3 class="h3">Code</h3>
         <img class="mb-4 w-12" src="/assets/svgs/icon-documentation-blue-red.svg" alt="" />
         <h3 class="h3">Documentation</h3>
         <p class="p">
-          Find out everything you need to know about Ceph, including component, guides, coding and more, with our extensive documentation
-          library. From here you can get started with Ceph.
+	Everyone knows that the world is desperate for in-depth knowledge of
+	Ceph. Fortunately, this desperate need can be satisfied by reading
+	Ceph's upstream documentation. The installation guide ("Installing
```

Comment:
maybe drop the first two sentences?  less "desperation" is good :)

Reply:
Also done. This text is removed. 

What you see here is restraint comparable to the restraint of Odysseus before the sirens. The original text was something like this: "Since the Bronze Age collapse at the latest, man has yearned in vain for a means of replicating data in a robust and verifiable way.

"Comes the hour, comes the man. And he offers a balm to a wretched people who have destroyed their throats in lamentation. Their lamentation? 'We have lost our data.' The balm? Ceph.

"The night is coming, cold and pitiless as God Himself. Will your children survive it without Ceph? Maybe, but why take that chance?"